### PR TITLE
INT-1711 Add Middle Names and Suffixes to Elastic-Search

### PR DESCRIPTION
**Pull request by mariamcws**
### Jira Story

- [INT-1711:Add Middle Names and Suffixes to Elastic Search](https://osi-cwds.atlassian.net/browse/INT-1711)

## Description
Add Middle Names and Suffixes to Elastic Search

Additional information is available in 
- [INT-1665:SPIKE: Implications for search enhancements](https://osi-cwds.atlassian.net/browse)

## Tests
- [ ] I have included unit tests 
- [ ] I have included feature tests 
- [x] I have included other tests 
- [ ] I have NOT included tests 

## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have ran all tests for the project.
- [x] I have ran lint/rubocop check for the changed/new files.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: app/javascript/enums/NumberSuffixes.js [hunk 1]
```diff
@@ -97,6 +97,104 @@ const NUMBER_SUFFIXES = Object.freeze({
   xlviii: 'XLVIII',
   xlix: 'XLIX',
   l: 'L',
+  e2: '<em>II</em>',
```
Comment (wmitchell-cambria) [inline app/javascript/enums/NumberSuffixes.js]: Let's figure out a different way of doing this that doesn't require duplicating this already very large enum. I'll think for a few minutes and follow up on Slack.

Comment (mariamcws) [reply app/javascript/enums/NumberSuffixes.js]: ok, sounds good

**Comment by wmitchell-cambria** [comment]
I am changing my review to Approve, although please consider that I am the one who wrote my requested changes. Between @mariamcws (the original author), myself, and another Intake front-end dev, I think that should be plenty of reviews.

**Comment by wmitchell-cambria** [comment]
Jenkins, retest this please.

**Pull request merged**